### ISSUE TITLE
Doubt in constructor for KS0108 Controller based 128X64 GLCD

### ISSUE THREAD

**Issue by sudheerbussa** [olikraus/u8g2#2444]
My 128X64 GLCD has two Chip Select Pins (CS1,CS2), Now my question is inside constructor , Should i initialize CS2 to "NONE", or any other option. Because physically CS2 is not present.
U8G2_KS0108_128X64_F(rotation, d0, d1, d2, d3, d4, d5, d6, d7, enable, dc, cs0, cs1, cs2 [, reset])

**Comment by olikraus** [comment]
Yes, see the constructor here: https://github.com/olikraus/u8g2/wiki/gallery#15-dec-2016-ks0108-128x64-lcd
